packages/dropdowns/src/elements/Dropdown/Dropdown.tsx

```diff
@@ -219,7 +211,6 @@ const Dropdown: React.FunctionComponent<IDropdownProps & ThemeProps<DefaultTheme
             case Downshift.stateChangeTypes.mouseUp:
             case Downshift.stateChangeTypes.keyDownSpaceButton:
             case Downshift.stateChangeTypes.blurButton:
```

Comment:
I was on the fence about clearing the `input` when the button is blurred, so chose to keep parity with v7.